hdf5-types/Cargo.toml

```diff
@@ -8,13 +8,18 @@ description = "Native Rust equivalents of HDF5 types."
 repository = "https://github.com/aldanor/hdf5-rust"
 homepage = "https://github.com/aldanor/hdf5-rust"
 edition = "2018"
+build = "build.rs"
 
 [features]
-const_generics = []
+const-generics = []
+force-h5-allocator = []
 
 [dependencies]
 ascii = "1.0"
 libc = "0.2"
+hdf5-sys = { version = "0.7.1", path = "../hdf5-sys" }
```

Comment:
Add a `# !V` here? Try grepping for `0.7.1`, all lines with it should have this tag. There's a shell script that helps bumping all the versions at once.